scripts/build-package.mjs

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* eslint-env node */
+
+import { createRequire } from 'module';
+import upath from 'upath';
+import { build } from '@ckeditor/ckeditor5-dev-build-tools';
+
+( async () => {
```

Comment:
No need for IIFE and `async`. Top-level await is supported since Node 14.

Reply:
I used IIFE, because eslint is not happy about it top-level await

Reply:
Top-level await support was added in eslint 8 with `ecmaVersion: 2022` or higher. We are currently using eslint 7 and `ecmaVersion: 2020` and upgrading would require a lot of work that we currently don't have the capacity for.

Source: https://eslint.org/blog/2021/06/whats-coming-in-eslint-8.0.0/#support-for-es2022-class-fields%2C-top-level-await%2C-and-regexp-matching-indices

Reply:
Please report this as a new issue. We should handle it soon.

Reply:
https://github.com/ckeditor/ckeditor5/issues/16258